src/components/RangeSelector.js

```diff
@@ -92,6 +92,68 @@ class RangeSelector extends React.Component {
     });
   }
 
+  _handleClickStartOnTrack (e) {
+    const clientX = e.touches ? e.touches[0].clientX : e.clientX;
```

Comment:
Is the touches array supported in all browsers?  Maybe link to the documentation if you would.